src/internal/direction.ts

```diff
@@ -0,0 +1,12 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export function isRtl(element: HTMLElement | EventTarget) {
+  let isRtl = false;
+
+  if (element instanceof HTMLElement) {
+    isRtl = getComputedStyle(element).direction === 'rtl';
+  }
+
+  return isRtl;
```

Comment:
Are there any possible issues with incorrect default value on global hotkeys (if event target is `window`), which always assumes no-RTL?

Reply:
That's an interesting question. Off the top of my head, any callers of this function should be a deliberate request for the computed style of a specified element (and never the window). I'm actually updating the function to only take an Element because that satisfies a standard element, event.currentTarget, and various SVG node types. I checked it out and this out and Typescript will scream if it gets a window object.

Reply:
If I type it as `HTMLElement` then when I have `SVGCircleElement` the function will fail to return RTL as true. Any suggestions here?

Reply:
I'll set it back to `HTMLElement` for the scope of this PR so we can get it merged. When I handle the popover and charts we can continue the conversation on typing as I get a more granular picture of what we need to support.

Reply:
OK, makes sense to me to inly support what is needed at the moment.

Reply:
Unresolving this. @DaemonCahill you said you will set it back to `HTMLElement` but this did not happen